python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -152,6 +159,15 @@ export default function EditorContainer({
 
       dispatch(action);
 
+      const onError = (err: unknown) => {
+        const message = (err as RequestCallbackError).message ?? null;
```

Comment:
Is this guaranteed to be `RequestCallbackError`? Or should this be the same as the onChangePromptName's `onError`:
```
const message = err instanceof Error ? err.message : null;
```

Reply:
When the flask server returns an error, it results in a caught error but the actual value caught is a raw object with message -- it's not an instanceof Error. Since Error instances can also have message, this should handle both cases